rpcs3/Emu/Cell/Modules/cellRec.cpp

```diff
@@ -176,7 +176,8 @@ class rec_video_sink : public utils::video_sink
 	{
 		cellRec.notice("Stopping video sink. flush=%d", flush);
 
-		std::lock_guard lock(m_mtx);
+		std::lock_guard lock_video(m_video_mtx);
```

Comment:
std::scoped_lock provides optimizations for nested lockings.

Reply:
done